code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -454,10 +454,20 @@
 /datum/reagent/beer2/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
 	switch(current_cycle)
-		if(1 to 50)
-			M.Sleeping(4 SECONDS)
+		if(1 to 5)
+			if(prob(25))
+				M.emote("yawn")
+		if(6 to 9)
+			M.AdjustEyeBlurry(10 SECONDS)
+			if(prob(35))
+				M.emote("yawn")
+		if(10)
+			M.emote("faint")
+			M.Weaken(4 SECONDS)
+		if(11 to 50)
+			M.Paralyse(4 SECONDS)
 		if(51 to INFINITY)
-			M.Sleeping(4 SECONDS)
+			M.Paralyse(4 SECONDS)
```

Comment:
why not do 11 to INFINITY

Reply:
If I only changed the `if(11 to 50)` to `if(11 to INFINITY)`, the code would never move onto the part where toxin damage is applied, leaving the user paralysed and alive until the reagent leaves the system.

If we only had a single `if(11 to INFINITY)` statement that both the paralyse and adjusttoxinloss were in, then this line:

`update_flags |= M.adjustToxLoss((current_cycle - 50)*REAGENTS_EFFECT_MULTIPLIER, FALSE)`

Would cause the user to rapidly heal toxin damage from ticks 11 to 49. I could change that 50 to a 10 if you wanted the toxin to start ticking from the moment the user goes unconscious... but that's a pretty significant balance change, considering how quickly beer2's toxin damage scales. I'd like to make sure that's what you're asking for before I commit to it, thanks.